docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -0,0 +1,155 @@
+# Tizen Window System Shell
+
+
+The TZSH is an interface for manipulating windows of the sytem GUI services such as the quickpanel and the softkey. It provides C libraries that allow you to control windows of system GUI services and get notifications about the state changes of each window within an application.
```

Comment:
sytem  -> system

Reply:
Thanks.

Reply:
@gl-lee you have not corrected it yet, kindly do

Reply:
@gl-lee  thanks